<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>893</width>
    <height>633</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="widget" native="true">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>871</width>
      <height>491</height>
     </rect>
    </property>
    <widget class="QPushButton" name="submit">
     <property name="geometry">
      <rect>
       <x>170</x>
       <y>450</y>
       <width>181</width>
       <height>28</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <property name="text">
      <string>GÖNDER</string>
     </property>
    </widget>
    <widget class="QLabel" name="question">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>20</y>
       <width>701</width>
       <height>71</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <property name="text">
      <string>SORU</string>
     </property>
    </widget>
    <widget class="QWidget" name="verticalLayoutWidget">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>140</y>
       <width>311</width>
       <height>291</height>
      </rect>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QRadioButton" name="option_a">
        <property name="text">
         <string>A</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="option_b">
        <property name="text">
         <string>B</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="option_c">
        <property name="text">
         <string>C</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="option_d">
        <property name="text">
         <string>D</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="option_e">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="text">
         <string>E</string>
        </property>
        <property name="checkable">
         <bool>true</bool>
        </property>
        <property name="checked">
         <bool>false</bool>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
    <widget class="QFrame" name="frame">
     <property name="geometry">
      <rect>
       <x>440</x>
       <y>100</y>
       <width>361</width>
       <height>331</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QFrame{
border-radius:15px;
}</string>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
    </widget>
    <widget class="QLabel" name="id_text">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>81</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </widget>
   <widget class="QPushButton" name="sign_in">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>550</y>
      <width>151</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string>GİRİŞ</string>
    </property>
   </widget>
   <widget class="QLabel" name="info">
    <property name="geometry">
     <rect>
      <x>570</x>
      <y>540</y>
      <width>311</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>BİLGİ:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="enter_id">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>550</y>
      <width>161</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>893</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
